Marlin/ultralcd.cpp

```diff
@@ -4393,11 +4393,11 @@ void pad_message_string() {
     // pad with spaces to fill up the line
     while (j++ < LCD_WIDTH) lcd_status_message[i++] = ' ';
     // chop off at the edge
-    lcd_status_message[--i] = '\0';
+    lcd_status_message[LCD_WIDTH] = '\0';
```

Comment:
Shouldn't this be be `lcd_status_message[i] = '\0';` since `lcd_status_message` may contain non-printable characters, and using `LCD_WIDTH` will result in the message being clipped?